### PR TITLE
Supervised Machine Learning Documentation

### PR DESCRIPTION
**Pull request by Ghost-141**
This documentation covers the topics from the course of _**Supervised Machine Learning Algorithm**_  under Machine Learning Specialization from Coursera. This includes detailed documentation of theories and algorithm related to Linear Regression, Classification and Functions related to these machine learning algorithm. Along with that, it also includes graph, equations, standard notation related to these algorithm.

### Topics on Linear Regression:

- Simple Linear Regression
- Multi Variable Linear Regression
- Polynomial Regression
- Cost Function, Gradient Descent, Learning Rate
- Feature Selection and Feature Scaling

### Topics on Classification:

- Classification problem using Linear Regression
- Logistic Regression/Binary Classification
- Types of Decision Boundary for Classification
- Simplification of Gradient Descent, Loss and Cost Function for Classification 

### Topics on Model Fitting

- Overfitting, Underfitting and Generalization
- Solution to Overcome Overfitting and Underfitting
- Regularization

### CODE REVIEW

**Review by WaterMenon09 — CHANGES_REQUESTED** [review]
fix description

**Review by WaterMenon09 — APPROVED** [review]
